routes/api/users.js

```diff
@@ -35,6 +35,34 @@ router.post('/', (req, res) => {
   });
 });
 
+  console.log('hit route put /editItem');
+  if (req.user) {
+    //edit items
+    const modifiedItem = {
+      name: req.body.name,
+      description: req.body.description,
+      price: req.body.price
+    };
+    //add items to document
+    User.findOneAndUpdate(
+      { username: req.user.username },
+      { $pull: { items: { _id: req.body.itemId } } }, { safe: true, upsert: true }).then(
+        User.findOneAndUpdate(
+          { username: req.user.username },
+          { $push: { items: modifiedItem } },
+          { safe: true, upsert: true, new: true, runValidators: true }
+        ).then(dbItems => {
+          res.json(dbItems);
+        }).catch(err => {
+          res.json(err);
+        }).catch(err => {
+          res.json(err);
+        }));
+  } else {
+    res.status(404).json({ msg: 'NO SELLER LOGGED IN' });
+  }
+});
+
```

Comment:
did you mean to add all of this to the "/" route?

Reply:
I am not able to sign in on local, I think the additions added here should be removed. the edit route is still later in this file

Reply:
no i didnt, I tried to delete it then re push

Reply:
thanks! deleting this fixed the issues I was finding while testing 👍